packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -222,15 +222,16 @@ Media.createMediaConnection = (
         audio: audioStream?.outputStream?.getTracks()[0],
         video: videoStream?.outputStream?.getTracks()[0],
         screenShareVideo: shareVideoStream?.outputStream?.getTracks()[0],
+        // @ts-ignore
         screenShareAudio: shareAudioStream?.outputStream?.getTracks()[0], // TODO: add type for screenShareAudio in internal-media-core SPARK-446923
```

Comment:
For this issue, I have to add `@ts-ignore` because otherwise, types will not match. I see there is already a `TODO` for this issue, so I hope it will be fixed soon.